project/sites/ircommission/themes/ircommission_theme/src/scss/components.scss

```diff
@@ -188,7 +188,7 @@ a[href="/news"] {
 
 
 .search-form input[type="submit"] {
```

Comment:
I think all of this styling is provided by unity theme and shouldn't need to be specified at all in an individual site.